src/Block/AdminListBlockService.php

```diff
@@ -54,7 +54,7 @@ public function execute(BlockContextInterface $blockContext, ?Response $response
             }
         }
 
-        return $this->renderPrivateResponse($this->templateRegistry->getTemplate('list_block'), [
```

Comment:
anyone knows why we called `renderPrivateResponse` here? :man_shrugging: Is it important here to enforce a non-cacheable response?

Reply:
I don't see a real one